share/teoBase/scripts/teoBase.xml

```diff
@@ -38,14 +38,14 @@
     </module>
 
     <module>
-        <name>/sbin/runuser</name>
-        <parameters>-l teo -c "GST_PLUGIN_PATH=/usr/local/lib/gstreamer-1.0 speechRecognition.py"</parameters>
+        <name>su</name>
+        <parameters>- teo -c "GST_PLUGIN_PATH=/usr/local/lib/gstreamer-1.0 speechRecognition.py"</parameters> -->
```

Comment:
Does this actually work on the YARP module manager? That closing `-->` looks like malformed XML, and the fully expanded command resolves to `su - teo -c ...` (note the stray `-` character).

Reply:
yes, it works. See https://askubuntu.com/a/606149
> `su - <username> -c "<commands>"`
> `-` will simulate a login of the specified user
> -c tells it that you want to run a command

Reply:
No offense to the non-draconian error treatment, but the trailing ` -->` should definitely be removed.

Reply:
Upss! Sorry I didn't see the end. Now it's fixed :)
Thanks.